src/app/(main)/todos/[todoId]/note/create/page.tsx

```diff
@@ -34,64 +45,124 @@ export default function NotesPage() {
     setShowLink(true);
   };
 
+  // 로컬 스토리지에서 불러온 데이터를 toast에서 받은 경우, 모달을 연다.
+  const handleOpenLoadModal = (data: NoteData) => {
+    setLoadedNote(data);
+    setModalOpen(true);
+  };
+
+  // 모달에서 "불러오기" 버튼 클릭 시 실행
+  const handleLoadNote = (data: NoteData) => {
+    setTitle(data.title);
+    setContent(data.content);
+    if (data.link && data.link.trim() !== "") {
+      setLink(data.link);
+      setShowLink(true);
+    } else {
+      setLink("");
+      setShowLink(false);
+    }
+    setModalOpen(false);
+  };
+
+  // "임시저장" 버튼 클릭 시 TempSaveManager의 저장 함수 호출
+  const handleTempSave = () => {
+    tempSaveManagerRef.current?.saveTempNote();
+  };
+
   return (
-    <div className="ml-16 mt-24 md:ml-24 lg:ml-80">
-      <div className="container-width">
-        {/* 헤더 */}
-        <div className="grid h-44 grid-cols-[162px_auto] items-center">
-          <h1 className="truncate font-semibold text-slate-900 md:text-lg">
-            노트 작성
-          </h1>
-          <div className="flex justify-end gap-8">
-            <Button variant="outlined" size="sm" shape="square">
-              임시저장
-            </Button>
-            <Button variant="solid" size="sm" shape="square">
-              작성 완료
-            </Button>
+    <div className="min-h-screen bg-white">
+      <div className="ml-16 pt-24 md:ml-24 lg:ml-80">
+        <div className="container-width">
+          {/* 헤더 */}
+          <div className="grid h-44 grid-cols-[162px_auto] items-center">
+            <h1 className="truncate font-semibold text-slate-900 md:text-lg">
+              노트 작성
+            </h1>
+            <div className="flex justify-end gap-8">
+              <Button
+                variant="outlined"
+                size="sm"
+                shape="square"
+                className="border-0"
+                onClick={handleTempSave}
+              >
+                임시저장
+              </Button>
+              <Button variant="solid" size="sm" shape="square">
+                작성 완료
+              </Button>
```

Comment:
피그마 보니 노트의 제목과 내용이 입력되지 않은 상태에서는 버튼이 `disabled` 처리되어 있는 것 같습니다! 다음과 같이 수정할 수 있을 것 같아요

```suggestion
              <Button variant="solid" size="sm" shape="square" disabled={!title.trim() || !content.trim()}>
                작성 완료
              </Button>
```

Reply:
나중에 api 연결 시 추가하려고 했던 부분인데, 발견해주셔서 지금 수정했습니다! 감사합니다.